### ISSUE TITLE
Connection diagram

### ISSUE THREAD

**Issue by ranawaqas3051** [PatrickSuhm/SelfDrivingGPSRobot#3]
Sir. Great work done by you on selfdriving GPS Robot. But i need pin description or connection diagram of that project. Please

**Comment by LeighDavis** [comment]
Hi @ranawaqas3051, were you able to figure out the pin connections? If yes, can you share it with us?

@PatrickSuhm Can you please share it if possible? Thanks!

**Comment by ranawaqas3051** [comment]
No. Bro... Sent from my Huawei Mobile-------- Original Message --------Subject: Re: [PatrickSuhm/SelfDrivingGPSRobot] Connection diagram (#3)From: LeighDavis To: PatrickSuhm/SelfDrivingGPSRobot CC: ranawaqas3051 ,Mention 
Hi @ranawaqas3051, were you able to figure out the pin connections? If yes, can you share it with us?
@PatrickSuhm Can you please share it if possible? Thanks!

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/PatrickSuhm/SelfDrivingGPSRobot/issues/3#issuecomment-811993936",
"url": "https://github.com/PatrickSuhm/SelfDrivingGPSRobot/issues/3#issuecomment-811993936",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by pro3088** [comment]
if possible can we get the schematic diagram of your connection or a video showing how the components are connected, i have some difficulty recreating your splendid project
please i'll appreciate this when you have the time to respond
